src/gwt/www/signin.css

```diff
@@ -192,3 +193,14 @@ div.signinhidden {
 .signin-hidden {
   display: none;
 }
+
+:focus {
+  outline: none !important;
+  box-shadow: none !important;
```

Comment:
`!important` :(

 But sometimes you have to do what you must.

Reply:
I copy and pasted this directly from your focus-visible work on the home page. :-)